apps/Dataman/babel.config.js

```diff
@@ -4,6 +4,21 @@ module.exports = function (api) {
     presets: ["babel-preset-expo"],
     plugins: [
       "nativewind/babel",
+      [
+        "module:react-native-dotenv",
+        {
+          envName: "APP_ENV",
```

Comment:
?

Reply:
It allows to use environment variables in React Native application.

Reply:
put a reference link to the blog/docs you followed. Put it as a comment in the code at this position as well as here.

Reply:
https://docs.expo.dev/guides/environment-variables/     
https://blog.logrocket.com/understanding-react-native-env-variables/ 
These are the link of expo documentation and blog from where I have taken a reference.